docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -9,25 +9,23 @@
    "source": [
     "# Introduction\n",
     "\n",
-    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakeFS-spec` for data versioning. We will use weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
+    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakefs-spec` for data versioning. We will use versioned weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
     "\n",
     "We will do the following:\n",
     "\n",
     "* Environment setup\n",
-    "* lakeFS setup\n",
-    "* Data ingestion\n",
-    "    * Transactions\n",
-    "    * PUT a file\n",
+    "* LakeFS setup\n",
```

Comment:
I propose to skip this section (not to repeat quickstart doc)

Reply:
So I'm wondering at what stage we assume the user to check this notebook? If the user does not look at the quickstart initially, he is left without any instructions. I think it's worth including some setup in both - what amount that should be can be discussed.

Reply:
Up until now, we have always had the imperative that the Demo notebook should work as a standalone unit. If we want to continue with this, we should keep the section. 
I prefer this approach as I, as a user, would expect a demo to be self-contained. 

If we decide against self-containment, we could significantly trim the set-up explanation as well.